qa-tests-backend/src/test/groovy/RetryTest.groovy

```diff
@@ -0,0 +1,22 @@
+import util.Retry
+
+import spock.lang.Shared
+import spock.lang.Specification
+
+class RetryTest extends Specification {
+    def test() {
+        expect:
+        fail("called", 1)
+    }
+
+    @Shared
```

Comment:
Is `@Shared` really needed here? Does this mean that retries of the same feature method considered different methods?!